tsconfig.json

```diff
@@ -1,8 +1,11 @@
 {
+  "exclude": ["node_modules"],
   "compilerOptions": {
+    "strict": true,
     "target": "ESNext",
     "module": "ESNext",
-    "moduleResolution": "nodenext",
+    "moduleResolution": "Node",
```

Comment:
`nodenext` is a bit early, breaks various things, e.g. any imports from `@tabler/icons`